.php-cs-fixer.cache

```diff
@@ -0,0 +1 @@
+{"php":"8.1.9","version":"3.10.0","indent":"    ","lineEnding":"\n","rules":{"blank_line_after_namespace":true,"braces":true,"class_definition":true,"constant_case":true,"elseif":true,"function_declaration":true,"indentation_type":true,"line_ending":true,"lowercase_keywords":true,"method_argument_space":{"on_multiline":"ensure_fully_multiline","keep_multiple_spaces_after_comma":true},"no_break_comment":true,"no_closing_tag":true,"no_space_around_double_colon":true,"no_spaces_after_function_name":true,"no_spaces_inside_parenthesis":true,"no_trailing_whitespace":true,"no_trailing_whitespace_in_comment":true,"single_blank_line_at_eof":true,"single_class_element_per_statement":{"elements":["property"]},"single_import_per_statement":true,"single_line_after_imports":true,"switch_case_semicolon_to_colon":true,"switch_case_space":true,"visibility_required":{"elements":["method","property"]},"encoding":true,"full_opening_tag":true,"array_syntax":{"syntax":"short"},"ordered_imports":{"sort_algorithm":"alpha"},"no_unused_imports":true,"not_operator_with_successor_space":true,"trailing_comma_in_multiline":true,"phpdoc_scalar":true,"unary_operator_spaces":true,"binary_operator_spaces":true,"blank_line_before_statement":{"statements":["break","continue","declare","return","throw","try"]},"phpdoc_single_line_var_spacing":true,"phpdoc_var_without_name":true,"class_attributes_separation":{"elements":{"method":"one"}},"single_trait_insert_per_statement":true},"hashes":{"src\/Actions\/ActionInterface.php":3794870253,"src\/Actions\/ClearCacheAction.php":3391253236,"src\/LaravelStoryblokFacade.php":465150087,"src\/Http\/Controllers\/StoryblokWebhookController.php":936241491,"src\/Http\/Middleware\/VerifyStoryblokWebhookSignature.php":2251817492,"src\/Commands\/ClearStoryblokCacheCommand.php":452607207,"src\/LaravelStoryblokServiceProvider.php":2359557763,"src\/Storyblok.php":3594343834,"tests\/StoryblokTest.php":1140628678,"tests\/Http\/Controllers\/StoryblokWebhookControllerTest.php":2828080726,"tests\/Commands\/ClearStoryblokCacheCommandTest.php":2242150547,"tests\/Stubs\/WebhookActionWithoutInterface.php":1709632369,"tests\/TestCase.php":3588041990}}
```

Comment:
This file should not be in git, add it to gitignore

Reply:
done